test/stripe/api_resource_test.rb

```diff
@@ -385,7 +408,19 @@ class NestedTestAPIResource < APIResource
         c.save
       end
 
-      should "add key to nested objects" do
+      should "passing in a stripe_account header should pass it through on update" do
+        stub_request(:get, "#{Stripe.api_base}/v1/customers/cus_123")
+          .with(headers: { "Stripe-Account" => "acct_123" })
+          .to_return(body: JSON.generate(customer_fixture))
+        Stripe::Customer.retrieve("cus_123", stripe_account: "acct_123")
+
+        stub_request(:post, "#{Stripe.api_base}/v1/customers/cus_123")
+          .with(headers: { "Stripe-Account" => "acct_123" })
+          .to_return(body: JSON.generate(customer_fixture))
+        Stripe::Customer.update("cus_123", { description: "FOO" })
+      end
```

Comment:
No need to test this, `.save` has a relationship with the previous `.retrieve` but `Update` should not.
```suggestion
```